jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -57,6 +57,16 @@
  */
 // TODO: Add tests once containerize() is added.
 public class JibContainerBuilder {
+  /**
+   * Create a container for a given build result. For internal use only.
+   *
+   * @param imageDigest the digest of the registry image manifest
+   * @param imageId digest of the container configuration
+   * @return the new container
+   */
+  public static JibContainer created(DescriptorDigest imageDigest, DescriptorDigest imageId) {
```

Comment:
Yea, so for example related to https://github.com/GoogleContainerTools/jib/pull/1155#discussion_r227173561, here, it feels a bit off that `JibContainerBuilder` calls into `BuildSteps` to run the build and `BuildSteps` calls `JibContainerBuilder` to create its result.